ticdc/ticdc-avro-protocol.md

```diff
@@ -0,0 +1,260 @@
+---
+title: TiCDC Avro Protocol
+summary: Learn the concept of TiCDC Avro Protocol and how to use it.
+---
+
+# TiCDC Avro Protocol
+
+Avro is a data exchange format protocol defined by [Apache Avro™](https://avro.apache.org/) and chosen by [Confluent Platform](https://docs.confluent.io/platform/current/platform.html) as the default data exchange format. This document describes the implementation of the Avro data format in TiCDC, including TiDB extension fields, definition of the Avro data format, and the interaction between Avro and [Confluent Schema Registry](https://docs.confluent.io/platform/current/schema-registry/index.html).
+
+## Use Avro
+
+When using Message Queue (MQ) as downstream sink, you can specify Avro in `sink-uri`. TiCDC wraps Avro Messages with events as the basic unit and sends TiDB DML events downstream. When Avro detects a schema change, it registers the latest schema with Schema Registry.
```

Comment:
1. TiCDC captures TiDB DML events and creates Avro messages from them, send these events downstream.
2. register the latest schema into schema registry.

Reply:
> 1. TiCDC captures TiDB DML events and creates Avro messages from them, send these events downstream.
> 2. register the latest schema into schema registry.

1. accepted. Please synchronize the change to Chinese in https://github.com/pingcap/docs-cn/pull/9449.
2. "register with" is a fixed expression. See the picture captured from Confluent:
![image](https://user-images.githubusercontent.com/89768198/170263867-f1eed42a-b277-4ea6-94de-5b461163e6d4.png)

Reply:
Ohh, then it's fine.